checkouts_service.go

```diff
@@ -13,39 +13,21 @@ type CheckoutsService service
 // Create a custom checkout.
 //
 // https://docs.lemonsqueezy.com/api/checkouts#create-a-checkout
-func (service *CheckoutsService) Create(ctx context.Context, params *CheckoutCreateParams) (*CheckoutApiResponse, *Response, error) {
-	checkoutData := map[string]any{
-		"custom": params.CustomData,
-	}
-	if params.DiscountCode != nil {
-		checkoutData["discount_code"] = params.DiscountCode
-	}
-
+func (service *CheckoutsService) Create(ctx context.Context, variantId int, storeId int, attributes *CheckoutCreateAttributes) (*CheckoutApiResponse, *Response, error) {
```

Comment:
I see you used `int` for `variantId` and `storeId` instead of `string` I like the idea of int but this will make the library inconsistent, if only one endpoint requires int and the others require `string` I'll propose either keep as `string` or refactor all the other endpoints to use `int`


Another thing is the order of endpoints, usually i use the order of `(parent, child, grandchild)` so since `variant` is a property of the `storeId` it should come after store in the list of parameters.

```go
func (service *CheckoutsService) Create(ctx context.Context, storeID, variantID int, attributes *CheckoutCreateAttributes) (*CheckoutApiResponse, *Response, error) {
}